uxkit-youtube/src/main/java/com/voxeet/uxkit/youtube/YoutubePlayerViewListener.java

```diff
@@ -0,0 +1,64 @@
+package com.voxeet.uxkit.youtube;
+
+import androidx.annotation.NonNull;
+
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.PlayerConstants;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.YouTubePlayer;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.listeners.YouTubePlayerListener;
+import com.voxeet.uxkit.common.UXKitLogger;
+import com.voxeet.uxkit.common.logging.ShortLogger;
+
+public abstract class YoutubePlayerViewListener implements YouTubePlayerListener {
+
+    private final ShortLogger Log = UXKitLogger.createLogger(getClass());
+
+    @Override
+    public void onApiChange(@NonNull YouTubePlayer youTubePlayer) {
+        Log.d("onApiChange");
+    }
+
+    @Override
+    public void onCurrentSecond(@NonNull YouTubePlayer youTubePlayer, float v) {
+
```

Comment:
We should log something here or mark it as `// no-op`

Reply:
+1